pkg/cmd/ssh.go

```diff
@@ -138,12 +122,19 @@ func getSSHKeypair(shoot *gardencorev1alpha1.Shoot) *v1.Secret {
 }
 
 // printNodeNames print all nodes in k8s cluster
-func printNodeNames(kindIP string) {
+func printNodeNames(kindIP string, shoot *gardencorev1alpha1.Shoot) (err error) {
```

Comment:
The kindIP arg seems unused, can we remove it?
```suggestion
func printNodeNames(shoot *gardencorev1alpha1.Shoot) (err error) {
```